static/src/javascripts-legacy/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -78,23 +79,19 @@ define([
 
         function disableRadioButtons(buttonClassName) {
             var radioButtons = document.getElementsByClassName(buttonClassName);
```

Comment:
Sorry this is our fault for not explaining properly. You should actually be getting the elements like this: `$("." + buttonClassName')` and then use Bonzo methods. The `$` is a mixture of query and bonzo that we have in the codebase